helper-cli/src/main/kotlin/commands/repoconfig/GenerateScopeExcludesCommand.kt

```diff
@@ -214,6 +214,11 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 pattern = "test",
                 reason = ScopeExcludeReason.TEST_DEPENDENCY_OF,
                 comment = "Packages for testing only."
+            ),
```

Comment:
I just noticed that this scope was from a Gradle project, the the entry should not go under "Maven" but under "Gradle".

Reply:
done